dev-client/src/screens/SlopeScreen/SlopeMeterScreen.tsx

```diff
@@ -82,68 +84,74 @@ export const SlopeMeterScreen = ({siteId}: {siteId: string}) => {
   }, [dispatch, siteId, deviceTiltDeg, navigation]);
 
   return (
-    <ScreenScaffold AppBar={null} BottomNavigation={null}>
-      <Row flex={1} alignItems="stretch" px="24px" py="20px">
-        <Box flex={1} justifyContent="center">
-          {permission?.granted ? (
-            <Camera style={styles.camera}>
-              <Column flex={1} alignItems="stretch">
-                <Box flex={1} />
-                <Box height="3px" bg="text.primary" />
-                <Box height="3px" bg="primary.contrast" />
-                <Box flex={1} bg="#00000080" />
-              </Column>
-            </Camera>
-          ) : permission?.canAskAgain ? (
-            <Button size="lg" onPress={requestPermission}>
-              {t('slope.steepness.camera_grant')}
-            </Button>
-          ) : (
-            <>
-              <Heading variant="h6">{t('slope.steepness.no_camera')}</Heading>
-              <Text variant="body1" textAlign="center">
-                {t('slope.steepness.no_camera_explanation')}
-              </Text>
-              <Link onPress={Linking.openSettings}>
-                {t('general.open_settings')}
-              </Link>
-            </>
-          )}
-        </Box>
-        <Column alignItems="center">
-          <Box {...styles.closeButtonBox}>
-            <CardCloseButton size="lg" onPress={onClose} />
+    <>
+      <ScreenScaffold AppBar={null} BottomNavigation={null}>
+        <Row flex={1} alignItems="stretch" px="24px" py="20px">
+          <Box flex={1} justifyContent="center">
+            {permission?.granted ? (
+              <Camera style={styles.camera}>
+                <Column flex={1} alignItems="stretch">
+                  <Box flex={1} />
+                  <Box height="3px" bg="text.primary" />
+                  <Box height="3px" bg="primary.contrast" />
+                  <Box flex={1} bg="#00000080" />
```

Comment:
suggestion: give this color a name and put it in theme.ts.

Reply:
I'll make a note to do this in a refactoring task, since this is unrelated to the change (same deal as above where the diff is misleading because I just added something to the hierarchy)